### PR TITLE
Get PSU status from STATE_DB

### PR DESCRIPTION
**Pull request by andriymoroz-mlnx**
Signed-off-by: Kevin Wang <kevinw@mellanox.com>

**- What I did**

**- How I did it**

**- How to verify it**

**- Previous command output (if the output of a command-line utility has changed)**

**- New command output (if the output of a command-line utility has changed)**

-->

### CODE REVIEW

File: show/main.py [hunk 1]
```diff
@@ -1091,12 +1091,42 @@ def syseeprom(verbose):
...
-        cmd += " -i {}".format(index)
+    db = SonicV2Connector(host="127.0.0.1")
+    db.connect(db.STATE_DB)
+
+    # Currently set chassis_num to 1, need to improve it once new platform API is implemented
+    chassis_num = 1
+    chassis_name = "chassis{}".format(chassis_num)
+    num_psus = db.get(db.STATE_DB, 'CHASSIS_INFO|{}'.format(chassis_name), 'num_psus')
+    if not num_psus:
+        click.echo("Error! PSU daemon is broken!")
```
Comment (andriymoroz-mlnx) [inline show/main.py]: message is too generic :)
I'd suggest something closer to the truth, like "Failed to get the number of PSUs"

File: show/main.py [hunk 2]
```diff
@@ -1091,12 +1091,42 @@ def syseeprom(verbose):
...
+    else:
+        psu_ids = [index]
+
+    header = ['PSU', 'Status']
+    status_table = []
+
+    for psu in psu_ids:
+        msg = ""
+        psu_name = "PSU{}".format(psu)
+        if psu not in supported_psu:
```
Comment (andriymoroz-mlnx) [inline show/main.py:1116]: I would move this validation to 1108
implementation looks better when it validates input first and then processes it

Comment (andriymoroz-mlnx) [reply show/main.py:1116]: oh, if what we get in num_psus is not equal to "actually installed"
message should be changed to something like "PSUX is not installed"

Comment (kevinwangsk) [reply show/main.py:1116]: The printing message here is just copied from psuutil, so shall we just keep as it is?

Comment (andriymoroz-mlnx) [reply show/main.py:1116]: ok, just noticed user can specify the PSU index.
in such case message makes sense.

Comment (kevinwangsk) [reply show/main.py:1116]: Yes, thanks for the comments all the same. :)

**Pull request closed without merging**